command/agent/job_endpoint_test.go

```diff
@@ -4356,3 +4356,51 @@ func Test_apiWorkloadIdentityToStructs(t *testing.T) {
 		TTL:          2 * time.Hour,
 	}))
 }
+
+func TestConversion_ApiJobUIConfigToStructs(t *testing.T) {
+	t.Run("nil jobUI", func(t *testing.T) {
+		require.Nil(t, ApiJobUIConfigToStructs(nil))
```

Comment:
Use `must.Nil`/`must.Eq` on new tests, instead of `require`. We're slowing trying to migrate over to use `shoenig/test` instead of `testify`